project/.travis/before_install_test.sh.twig

```diff
@@ -14,6 +14,10 @@ fi
 composer self-update --stable
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
+{% if 'mongodb' in services %}
+if [ "$MONGODB_VERSION" != "" ]; then pecl install -f mongodb-$MONGODB_VERSION; fi
```

Comment:
Please use `--force` instead of `-f`, it's more explicit. Also, why do you need this option?